lua/class_definitions.lua

```diff
@@ -1130,7 +1130,7 @@ classes = {
             { name = "spawn_item", rval = nil, args = { "tripoint", "string", "int", "int" } },
             { name = "spawn_item", rval = nil, args = { "tripoint", "string", "int", "int", "int" } },
             { name = "spawn_item", rval = nil, args = { "tripoint", "string", "int", "int", "int", "int" } },
-            { name = "spawn_item", rval = nil, args = { "tripoint", "string", "int", "int", "int", "int", "bool" } },
+            { name = "spawn_item", rval = nil, args = { "tripoint", "string", "int", "int", "int", "int" } },
```

Comment:
This line looks like the same above it, there is no need to have this line appear twice, remove one of them.